test/parse_test.cc

```diff
@@ -0,0 +1,89 @@
+#include <gtest/gtest.h>
+
+#include <vector>
+
+#include "xcshell/parser.h"
+
+TEST(ParseTest, should_prase_output_redirection_correctly_with_overwrite) {
```

Comment:
Test case name do not follow our style, check out other test file for reference.